Source/Editor/Options/InterfaceOptions.cs

```diff
@@ -138,6 +138,27 @@ public enum ValueFormattingType
             AutoUnit,
         }
 
+        /// <summary>
+        /// Options for on play mode start panel focus.
+        /// </summary>
+        public enum PlayModeFocus
+        {
+            /// <summary>
+            /// Don't change any focus.
+            /// </summary>
+            None,
+
+            /// <summary>
+            /// Focus the Game panel.
+            /// </summary>
+            GamePanel,
```

Comment:
Use `GameWindow` name to match naming in Editor.

Reply:
Added in https://github.com/FlaxEngine/FlaxEngine/pull/2994/commits/2721364df6be1c5d38ebcf1055ebbc2beda5863e.